.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Captum Deploy Website
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
+
+jobs:
+  tests:
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      docker-image: cimg/python:3.6-node
+      repository: pytorch/captum
+      script: |
+        sudo chmod -R 777 .
+        ./scripts/install_via_pip.sh -n -d
```

Comment:
For any calls to `pip install` within this script or any other script called from these workflows, we probably should add the following option to make the logs more readable: `--progress-bar off`